android/build.gradle

```diff
@@ -7,6 +7,7 @@ buildscript {
         gradlePluginPortal()
     }
     dependencies {
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${safeExtGet("kotlinVersion", "1.8.0")}")
         classpath("com.android.tools.build:gradle:7.3.1")
```

Comment:
Maybe a Version catalogue would be nice. Also If you only want to support 2 major versions, Then RN 0.73 technically shipped with [AGP 8.1.1](https://github.com/facebook/react-native/blob/v0.73.9/packages/react-native-gradle-plugin/gradle/libs.versions.toml). Could be fun having a higher target

Reply:
Good catch! Bumped to 8.1.1